### PR TITLE
KOGITO-3589 Extracted kogito-apps from runtimes

### PR DESCRIPTION
**Pull request by radtriste**
https://issues.redhat.com/browse/KOGITO-3589

Dependent PRs:
- https://github.com/kiegroup/kogito-pipelines/pull/89
- https://github.com/kiegroup/kogito-runtimes/pull/909
- https://github.com/kiegroup/kogito-apps/pull/554

### CODE REVIEW

File: Jenkinsfile.nightly [hunk 1]
```diff
@@ -26,9 +27,9 @@ pipeline {
         label 'kie-rhel7 && !master'
     }
 
-    triggers {
-        cron ('H 0 * * *')
-    }
+    // triggers {
+    //     cron ('H 0 * * *')
+    // }
```
Comment (radtriste) [inline Jenkinsfile.nightly]: for testing purpose. will be set back

File: Jenkinsfile.release [hunk 2]
```diff
@@ -66,25 +68,25 @@ pipeline {
...
+        GITHUB_TOKEN_CREDS_ID='radtriste-gh-token' // secret text creds
 
         // Maven Nexus release parameters
         DEFAULT_STAGING_REPOSITORY = 'https://repository.jboss.org/nexus/content/groups/kogito-public/'
-        NEXUS_RELEASE_URL = 'https://repository.jboss.org/nexus'
+        NEXUS_RELEASE_URL = 'https://repository.stage.jboss.org/nexus'
         NEXUS_RELEASE_REPOSITORY_ID = 'jboss-releases-repository'
         NEXUS_STAGING_PROFILE_ID = '2161b7b8da0080'
-        NEXUS_BUILD_PROMOTION_PROFILE_ID = 'ea49ccd6f174'
+        NEXUS_BUILD_PROMOTION_PROFILE_ID = '1966c60aff37d'
```
Comment (radtriste) [inline Jenkinsfile.release]: for testing purpose. will be set back

**Review by radtriste — COMMENTED** [review]
all commented part will be set back once testing is finished

**Comment by radtriste** [comment]
testing in progress

**Review by Kevin-Mok — APPROVED** [review]
I had to locally check out the branch and uncomment all the lines to see the actual diff. Perhaps it would be better to use another branch for testing? Otherwise, it looks good.

**Comment by radtriste** [comment]
> I had to locally check out the branch and uncomment all the lines to see the actual diff. Perhaps it would be better to use another branch for testing? Otherwise, it looks good.

@Kevin-Mok right sorry about that ... done it

**Review by rsynek — APPROVED** [review]

**Comment by rsynek** [comment]
Big thanks for this refactoring @radtriste !

**Comment by radtriste** [comment]
Testing ok https://docs.google.com/spreadsheets/d/1L4NvDuQ6h4nOVuXMgpVmlvF-sjoTH6eI6EJFLWiwe7I/edit?usp=sharing
@rsynek good to merge

**Pull request merged**